docs/data-sources/compute_instance_remote_console.md

```diff
@@ -0,0 +1,46 @@
+---
+subcategory: "Elastic Cloud Server (ECS)"
+---
+
+# huaweicloud_compute_instance_remote_console
+
+Use this data source to get available HuaweiCloud ECS compute instance remote console.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+
+data "huaweicloud_compute_instance_remote_console" "test" {
+    instance_id = var.instance_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) The region in which to obtain the ECS compute instance remote console.
+  If omitted, the provider-level region will be used.
+
+* `instance_id` - (Required, String) The ID of the ECS Cloud Server.
```

Comment:
please unify the description of ECS instance

Reply:
done